packages/strapi-utils/lib/content-types.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const _ = require('lodash');
+
+const getPrivateAttributes = (model = {}) => {
```

Comment:
If you remove the forEach above then you don't need this function and can optimize the isPrivateAttribute one :)